ui/pages/Public/components/FeatureUpdates.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react'
+import { Feed, Header } from 'semantic-ui-react'
+import TextFieldView from 'shared/components/panel/view-fields/TextFieldView'
+import { HttpRequestHelper } from 'shared/utils/httpRequestHelper'
+
+const FeatureUpdates = () => {
+  const [errorMsg, setErrorMsg] = useState('')
+  const [feedEntries, setFeedEntries] = useState([])
+
+  useEffect(() => {
```

Comment:
🤔 I thought we were on a version of react that didn't let these work... and I think we're using `this.setState` elsewhere... Hana can give more direction here I think!

Reply:
We discussed this offline. We do not really use react hooks in seqr because when we did the initial migration to using react for the frontend 5 years ago it wasn't supported, but the version we are on now does work with hooks, we just don't have the bandwidth to go back and do a full overhaul of the repo to use hooks instead of our big clunky redux state. There are pros and cons to keeping all the code in the repo consistent and never using hooks vs slowly migrating to using hooks and having some files use them and some not. But given that this is the more modern style and is more likely to be industry standard going forard, I am comfortable with us starting a gradual migration

Reply:
hooks also make sense here because the state values of FeatureUpdates are only accessed within the component and don't need to be a part of the global redux state